lms/static/js/learner_dashboard/models/program_progress_model.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Model for User's Program Status.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
```

Comment:
It's surprise to me we need to construct a backbone model this way. There is no good way to just create a blank and nameless backbone model, instead of creating a new file like this?